inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.hpp

```diff
@@ -54,8 +55,12 @@ class LayerTestsCommon : public CommonTestUtils::TestsCommon {
 
     virtual void Compare(const std::vector<std::uint8_t> &expected, const InferenceEngine::Blob::Ptr &actual);
 
+    virtual void Compare(const std::vector<InferenceEngine::Blob::Ptr>& expectedOutputs, const std::vector<InferenceEngine::Blob::Ptr>& actualOutputs);
```

Comment:
Do we really need to introduce a new `Compare` interface? For me, It makes more sense to change `computeCpuTarget` return value to `std::vector<uint8_t>` and use already existing method. I even guess it will require to write lesser code.